docs/glossary.rst

```diff
@@ -1155,11 +1155,9 @@ Glossary
    cookiecutter
       A command-line utility that creates projects from :ref:`cookiecutters <cookiecutter:readme>` (project templates), e.g., creating a Python package project from a Python package project template.
 
-      Pyramid cookiecutters include:
+      The Pyramid cookiecutter can be found here:
 
-      * `pyramid-cookiecutter-alchemy <https://github.com/Pylons/pyramid-cookiecutter-alchemy>`_
-      * `pyramid-cookiecutter-starter <https://github.com/Pylons/pyramid-cookiecutter-starter>`_
-      * `pyramid-cookiecutter-zodb <https://github.com/Pylons/pyramid-cookiecutter-zodb>`_
+      * `pyramid-cookiecutter-theonecc <https://github.com/Pylons/pyramid-cookiecutter-theonecc>`_
 
       .. versionadded:: 1.8
 
```

Comment:
We need to have a deprecation notice and versionadded notice. Let's use the following.

```rst
The unified cookiecutter ``pyramid-cookiecutter-theonecc`` merged features from the three previous cookiecutters which are now deprecated.

      * `pyramid-cookiecutter-theonecc <https://github.com/Pylons/pyramid-cookiecutter-theonecc>`_

      .. versionadded:: 1.10

      * `pyramid-cookiecutter-alchemy <https://github.com/Pylons/pyramid-cookiecutter-alchemy>`_
      * `pyramid-cookiecutter-starter <https://github.com/Pylons/pyramid-cookiecutter-starter>`_
      * `pyramid-cookiecutter-zodb <https://github.com/Pylons/pyramid-cookiecutter-zodb>`_

      .. deprecated:: 1.10

      .. versionadded:: 1.8
```

The reST syntax is tricky, so make sure indentation and extra lines don't break the docs build or result in wonky lists.

Reply:
I assumed that we wanted the onecc link prior to the `The unified cookiecutter pyramid-cookiecutter-theonecc merged features...` notice.

If that was incorrect, just let me know and I'd be happy to change it back.

Reply:
I think it reads better to remove and replace this line:

"The Pyramid cookiecutter can be found here:"

with this line:

"The unified cookiecutter ``pyramid-cookiecutter-theonecc`` merged features from the three previous cookiecutters which are now deprecated."

Also, as an FYI, since this is narrative documentation and not code, you don't have to wrap at 79 columns. It's OK (and preferred, although not enforced) to use one line per sentence.